### ISSUE TITLE
Test suite fails (with Git::Wrapper 0.045?)

### ISSUE THREAD

**Issue by eserte** [plicease-legacy/AnyEvent-Git-Wrapper#6]
The basic.t test fails on my smokers:

```
#   Failed test at t/basic.t line 124.
#     Structures begin differing at:
#          $got->{blob_from} = '0000000000000000000000000000000000000000'
#     $expected->{blob_from} = '0000000'
# Looks like you failed 1 test of 24.
t/basic.t ..........
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/24 subtests
```

Statistical analysis suggests that the failure is caused by Git::Wrapper 0.045; I have no fail reports with earlier Git::Wrapper versions...

**Comment by plicease** [comment]
Thanks for the report.  This should be fixed in 0.08, on its way to CPAN.

**Comment by eserte** [comment]
This looks good with Git::Wrapper 0.045, but fails now for older Git::Wrapper versions, so you want probably increase the minimum prereq version.

**Comment by plicease** [comment]
Totally right.  I had that in my mind when I was working on this release but then forgot to actually do it.  Thanks for following up on this :)  should be fixed for real this time (?) in 0.09 on its way to CPAN.

**Comment by eserte** [comment]
Looks good now!
